src/main/java/com/google/sps/data/TestAdaptor.java

```diff
@@ -0,0 +1,43 @@
+package com.google.sps.data;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+/** File to test the GetGovAdator file.
+ */
+public class TestAdaptor { 
```

Comment:
We don't have time for proper unit test, I believe this file will be deleted soon, right?

Reply:
yes, its just a test file for me to run local runs

Reply:
For your interest, JUnit is something we should try if there is time left